docker/auth_db/V1000_0__local_clients.sql

```diff
@@ -1,10 +1,15 @@
 -- Add auth client (auth_code flow) for local dev. Client secret is clientsecret
 INSERT INTO oauth_client_details (client_id, access_token_validity, additional_information, authorities, authorized_grant_types, autoapprove, client_secret, refresh_token_validity, resource_ids, scope, web_server_redirect_uri)
-VALUES ('hmpps-arns-assessment-platform-ui', 3600, '{}', null, 'authorization_code,refresh_token', 'read,write', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', 43200, null, 'read,write', 'http://localhost:3000/sign-in/callback,http://ui:3000/sign-in/callback');
+VALUES ('hmpps-arns-assessment-platform-ui', 3600, '{}', null, 'authorization_code,refresh_token', 'read,write', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', 43200, null, 'read,write', 'http://localhost:3000/sign-in/callback,http://localhost:3000/sign-in/hmpps-auth/callback,http://ui:3000/sign-in/callback,http://ui:3000/sign-in/hmpps-auth/callback');
 
 -- Add system client (S2S calls) for local dev with ARNS roles. Client secret is clientsecret
 INSERT INTO oauth_client_details (client_id, access_token_validity, additional_information, authorities, authorized_grant_types, autoapprove, client_secret, refresh_token_validity, resource_ids, scope, web_server_redirect_uri)
-VALUES ('hmpps-arns-assessment-platform-ui-system', 1200, '{}', 'ROLE_AAP__FRONTEND_RW', 'client_credentials', 'read,write', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', 43200, null, 'read,write', null);
+VALUES ('hmpps-arns-assessment-platform-ui-system', 1200, '{}', 'ROLE_AAP__FRONTEND_RW,ROLE_ARNS_ASSESSMENTS_RW,ROLE_ASSESS_RISKS_AND_NEEDS_COORDINATOR_RW', 'client_credentials', 'read,write', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', 43200, null, 'read,write', null);
+
+-- Add handover client for AAP (used by handover service). Client secret is aap-secret
+INSERT INTO oauth_client_details (client_id, access_token_validity, additional_information, authorities, authorized_grant_types, autoapprove, client_secret, refresh_token_validity, resource_ids, scope, web_server_redirect_uri)
+VALUES ('aap', 3600, '{}', null, 'authorization_code,refresh_token', 'read,write', '$2y$10$Juu9rW/BoxGMXO8ECkdESuaTZ7GhTzSLM1CFHC7ISZySxIgih7gDi', 43200, null, 'read,write', 'http://localhost:3000/sign-in/handover/callback,http://ui:3000/sign-in/handover/callback');
+
```

Comment:
I think this is adding a client to the HMPPS Auth DB - but the Handover service keeps track of its own clients separately. I think we've already added these into it's [configuration](https://github.com/ministryofjustice/hmpps-assess-risks-and-needs-handover-service/blob/012813ca1efc4e628ede6736c09f9f4d3df8b2b2/src/main/resources/application.yml#L117-L120)